packages/hooks/src/__tests__/useIsPasskeySupported.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { platformAuthenticatorIsAvailable } from '@simplewebauthn/browser';
+import useIsPasskeySupported from '../useIsPasskeySupported';
+
+jest.mock('@simplewebauthn/browser', () => ({
+    platformAuthenticatorIsAvailable: jest.fn(),
+}));
+
+describe('useIsPasskeySupported', () => {
```

Comment:
for the useIsPasskeySupported() - it is simply a wrapper hook around @simplewebauthn/browser which is an external dependencies. I think we shouldn’t test external dependencies